src/CalculatorUnitTests/RationalTest.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include <CppUnitTest.h>
+#include "Header Files/Rational.h"
+#include "Header Files/RationalMath.h"
+
+using namespace CalcEngine;
+using namespace CalcEngine::RationalMath;
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace CalculatorManagerTest
+{
+    TEST_CLASS(RationalTest)
+    {
+    public:
+        TEST_CLASS_INITIALIZE(CommonSetup)
+        {
+            ChangeConstants(10, 128);
+        }
+        TEST_METHOD(ModuloTest)
```

Comment:
Additional tests for both functions:

- Floating point values, positive and negative e.g. 1.5 mod 0.5, 1.5 mod 0.4.
- 0 as numerator
- Numerator and Denominator are the same
- Numerator and Denominator are equivalent but not the same e.g. pa = Rational(Number(27), Number(12)) and b = Rational(Number(9),Number(4))